ecs/variables.tf

```diff
@@ -65,3 +65,7 @@ variable "output_bucket" {
 variable "backend_client_secret_path" {
   default = ""
 }
+
+variable "vpc_id" {
+  default = ""
```

Comment:
Is it possible to run an ECS without a VPC? If not, perhaps there shouldn't be a default value? I think the Terraform error would be more obvious if the parameter is missing.

Reply:
Yep, good point, I've removed the default.